staging/src/k8s.io/apimachinery/pkg/runtime/converter.go

```diff
@@ -574,6 +574,9 @@ func interfaceFromUnstructured(sv, dv reflect.Value) error {
 func (c *unstructuredConverter) ToUnstructured(obj interface{}) (map[string]interface{}, error) {
 	var u map[string]interface{}
 	var err error
+	if obj == nil {
```

Comment:
maybe move the nil check before local variable definition is more clear.

Reply:
Done